blog/tests/models_test.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for Post model's Blog app."""
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from blog.models import Post
+
+
+class PostModelTest(TestCase):
+    """Tests for Post model."""
```

Comment:
Anything specific, except for copy-paste from module docstring?

Reply:
In top
```
"""Tests for models of Blog app."""
```
Doc's class
```
"""Tests for Post model."""
```

Reply:
Now I think it good. Maybe I still wrong.